e2e-test/src/test/resources/prod-config.yml

```diff
@@ -0,0 +1,8 @@
+# Placeholders for now
```

Comment:
We need a way for these YAML files to refer to environment variables in certain cases, such as in Prod.